custom-login/src/components/Login.vue

```diff
@@ -43,20 +43,16 @@ export default {
           }
         },
         authParams: {
-          pkce: true,
+          // To avoid redirect do not set "pkce" or "display" here. OKTA-335945
```

Comment:
Let's update these samples to use latest SIW version and call `showSignInToGetTokens` then this workaround will not be necessary